### PR TITLE
310673: Fork IPAFFS repo

### PR DESCRIPTION
**Pull request by reece-bennett**
Copy the IPAFFS mapping-library repository for reuse by STW. 

https://dev.azure.com/defragovuk/STW%20Integration/_workitems/edit/310673

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,46 +1,44 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <artifactId>Mapping-Library</artifactId>
+
+  <groupId>uk.gov.defra.stw</groupId>
+  <artifactId>mapping-library</artifactId>
   <version>4.0.1-SNAPSHOT</version>
```
Comment (pbControl) [inline pom.xml]: Is this supposed to be 4.0.1-SNAPSHOT?

Comment (reece-bennett) [reply pom.xml]: Changed all poms to 1.0.0

**Review by pbControl — DISMISSED** [review]

**Review by kamilmkainos — DISMISSED** [review]

**Review by joshuahigginson — DISMISSED** [review]
Looks good to me

File: README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# STW Mapping Library
```
Comment (subash-lakshmikanthan) [inline README.md:1]: can we please expand only at this place to give a view about the project? thanks.

**Review by kamilmkainos — APPROVED** [review]

**Review by joshuahigginson — APPROVED** [review]

**Review by subash-lakshmikanthan — APPROVED** [review]

**Review by pbControl — APPROVED** [review]

**Pull request merged**